### PR TITLE
FIX: dipole 3d plotting example shows dipole 6 as best dipole, however it's dipole 7

### PR DESCRIPTION
**Pull request by dengemann**
@agramfort @Eric89GXL do you agree that this is not ideal / confusing and should be adjusted?

### CODE REVIEW

File: mne/viz/_3d.py [hunk 1]
```diff
@@ -1931,7 +1931,7 @@ def _plot_dipole(ax, data, points, idx, dipole, gridx, gridy, ori, coord_frame,
                 zdir='y', cmap='gray', zorder=0, alpha=.5)
 
     plt.suptitle('Dipole %s, Time: %.3fs, GOF: %.1f, Amplitude: %.1fnAm\n' % (
-        idx, dipole.times[idx], dipole.gof[idx], dipole.amplitude[idx] * 1e9) +
+        idx + 1, dipole.times[idx], dipole.gof[idx], dipole.amplitude[idx] * 1e9) +
```
Comment (dengemann) [inline mne/viz/_3d.py]: we could alternatively write idx or index in the title.

**Comment by agramfort** [comment]
no strong feeling if it's readable.

Python user should be aware of zero indexing and I think we are pretty
consistent with this.

**Comment by dengemann** [comment]
> Python user should be aware of zero indexing and I think we are pretty
consistent with this.

I'm not seeing any consistence here. Dipole 6 is not Dipole 7 it's Dipole 6. If it is the index let's call it like this. Otherwise it makes you look up things at index 5.

File: mne/viz/_3d.py (hunk 1 above)
Comment (dengemann) [inline mne/viz/_3d.py]: @agramfort @Eric89GXL another option would be to use # as in ICA which seems to indicate index semantics in our plots (E.g. ```ICA #000```)

**Comment by agramfort** [comment]
or write Dipole[6] ?

**Comment by dengemann** [comment]
Yes ```Dipole[6]``` or ```Dipole [6]``` could work. Let's collect and see what is best. @mmagnuski what's your hunch on this. Some reminiscences of ICA index VS count semantics ...

**Comment by mmagnuski** [comment]
I don't have a strong opinion - `Dipole[6]` seems most straightforwardly indicating that it displays python index, but `#006` is also ok.

**Comment by agramfort** [comment]
we could also write:

Dipole 1 / 7
Dipole 6 / 7
etc.

**Comment by dengemann** [comment]
I like the last proposal. But Dipole[6] would be ok too.

**Comment by agramfort** [comment]
Pick your choice and let's merge

**Comment by larsoner** [comment]
@dengemann do you have time for this in the next week or so to make it into the next release?

**Comment by dengemann** [comment]
Will do later. Should take less then a minute.

**Comment by larsoner** [comment]
@dengemann can you get to this in the next few days or should someone else take over?

**Comment by larsoner** [comment]
@dengemann can you get to this in the next few days or should someone else take over? It's getting close to 0.15 release time

**Comment by larsoner** [comment]
Will takeover tomorrow unless I hear otherwise

**Comment by dengemann** [comment]
Sorry, was not available the last days. I kept away from this as I wanted to see how the ICA component display got resolved. I'd suggest to just copy the pattern from there.

**Comment by larsoner** [comment]
There is no equivalent pattern for ICA, as those are channel names (and are now referred to that way as often as possible IIRC).

**Pull request closed without merging**